### ISSUE TITLE
Shot Generator WebGL error?

### ISSUE THREAD

**Issue by fluffbirb** [wonderunit/storyboarder#692]
When I type in a shot, or toggle any of the shot generator options, this error pops up, and I can't get shot generator to do anything besides throw errors.

![image](https://user-images.githubusercontent.com/32004169/30575648-5684d6e8-9cd0-11e7-9e0c-b8dce7eaae3e.png)
https://imgur.com/a/GnMyY

Win7
Intel Core 2 Quad  CPU   Q8200  @ 2.33GHz, 2333 Mhz, 4 Core(s), 4 Logical Processor(s)
NVIDIA GeForce 9500 GT
8GB RAM

**Comment by fluffbirb** [comment]
Also the installer didn't give me a choice on where to install? My C: drive is rather small, and I prefer to install any non-OS stuff elsewhere.

**Comment by audionerd** [comment]
Thanks for the report. Not sure why the Shot Generator isn't working. We're using WebGL for the Shot Generator. It might be that your graphics card isn't fully supported by Electron/Chromium for WebGL with shaders.

Re: install location, I'll note your request in issue #698. Probably won't be a 1.0 thing but we'll take a look for 2.0

**Comment by fluffbirb** [comment]
I don't know if it's related, but it consistently happens:
I'm drawing on a board, and spontaneously the board clears while I'm drawing. I tried checking the images folder in case it still had my WIP drawing from the moment before it cleared, and those are all blank white as well. One time I found the blue pencil layer, but it was an older version.
I tried using the sprint feature to record my work, and that gets blanked out at the same time, as if the mark making never happened. Only the exports that finished before blanking out still exist.
Once it blanks out, program starts behaving oddly, sometimes doesn't receive input to draw.
After I lost my work 2-5 times, I quit using the program.
